README.md

```diff
@@ -132,6 +132,22 @@ httpd = require('http.server').new(host, port[, { options } ])
 
   By default uses `log.info` function for requests logging.
 * `log_errors` - same as the `log_requests` option but is used for error messages logging. By default uses `log.error()` function.
+* `keepalive_disable` - disables keep-alive connections with misbehaving
+  clients. Parameter accept a map that contains a user agents with non-nil
+  value. By default map is empty.
+
+  Example:
+
+  ```lua
+  local httpd = http_server.new('127.0.0.1', 8080, {
+      log_requests = true,
+      log_errors = true,
+      keepalive_disable = {
+          ['curl/7.68.0'] = true
+      },
```

Comment:
I don't know, whether it was already discussed or not, but `{['foo'] = true, ['bar'] = true}` is logical equivalent of `{'foo', 'bar'}`. I would expose the latter variant externally and transform it to the former internally.

I don't insist, but it looks a bit more accurate for me.

Reply:
I like idea. Patch updated.